src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] colors = new String[]{"blue", "yeLlo", "red", "purple", "green"};
+    private Random random = new Random();
+
+    public String setColor() {
```

Comment:
please use informative method names
```suggestion
    public String getRandomColor() {
```

Reply:
done